www/src/components/docs-breadcrumb.js

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import { Link } from "gatsby"
+import getActiveItem from "../utils/sidebar/get-active-item"
+import getActiveItemParents from "../utils/sidebar/get-active-item-parents"
+import { mediaQueries, space, fontSizes } from "../utils/presets"
+import { rhythm } from "../utils/typography"
+
+const Separator = ({ character = `>` }) => (
+  <span style={{ margin: `0px ${space[1]}` }} role="presentation">
+    {character}
+  </span>
+)
+
+const BreadcrumbNav = ({ children, mobile = false }) => (
+  <nav
+    aria-label="breadcrumb"
+    css={{
+      fontSize: fontSizes[1],
+      display: `${mobile ? `inherit` : `none`}`,
+      [mediaQueries.md]: {
+        display: `${mobile ? `none` : `inherit`}`,
+      },
+      marginBottom: rhythm(1 / 2),
```

Comment:
There's nothing wrong with still using `rhythm`, but I think we should stick to the design tokens.